CTP/doc/sql/sql_guide.md

```diff
@@ -0,0 +1,450 @@
+
+    * set 'Environment Variables'   
+      * After install cubrid by msi file, these system parameter will be added automatically:  
+		  ```
+		  CUBRID
+		  C:\CUBRID\
+
+		  CUBRID_DATABASES
+		  C:\CUBRID\databases
+		  ```
+      * Add new 'System variables':  
+		  ```
+		  JAVA_HOME
+		  C:\Program Files\Java\jdk1.8.0_201
+
+		  CTP_BRANCH_NAME
+		  develop
+
+		  CTP_SKIP_UPDATE
+		  0
+		  ```
+      * Edit 'path'  
+		  add '%JAVA_HOME%\bin C:\cygwin64\bin' in the 'path'.  
+
+    * Install CTP  
+      * Please follow the guide to install CTP.  
+      * touch ~/CTP/conf/sql_local.conf  
+		  Here is the config file that we use for current daily QA test: sql_local.conf  
+
+    * Check out test cases  
+		```
+		cd ~
+		git clone https://github.com/CUBRID/cubrid-testcases.git 
+		```
+    * touch start_test.sh    
+		```
+		start_test.sh
+		rm nohup.out
+		nohup start_consumer.sh -q QUEUE_CUBRID_QA_SQL_WIN64 -exec run_sql &
+		```
+		sh start_test.sh  
+# 3 Test run  
+## 3.1 manual test  
+### ctp  
+* run ctp.sh
+  ```
+  ctp.sh sql -c CTP/conf/sql_local.conf --interactive
+  ```
+  Modify the working directory and configuration parameters in the file CTP/conf/sql_local.conf\
+* run test
+  ```
+  ======================================  Welcome to Interactive Mode ======================================  
+  Usage: 
+      help         print the usage of interactive
+      run <arg>    the path of case file to run
+      run_cci <arg>    the path of case file to run_cci by cci driver
+      quit         quit interactive mode 
+
+  For example:
+      run .                                                 #run the current directory cases
+      run ./_001_primary_key/_002_uniq                      #run the cases which are based on the relative path
+      run test.sql                                          #run the test.sql file
+      run /home/user1/dailyqa/trunk/scenario/sql/_02_object #run the cases which are based on the absolute path
+      run_cci ./_001_primary_key/_002_uniq                  #run the cases which are based on the relative path
+      run_cci ./_001_primary_key/_002_uniq/test.sql         #run the cases file
+  sql>run .
+  ```
+
+  ![run_test](./ctp_run_test.png)  
+
+### csql  
+* sql test
+   * cubrid createdb testdb en_US  
+   * cubrid server start testdb
+   * csql -u dba testdb  
+  ![run_test](./csql_run_test.png)  
+   
+* medium test
+   * cubrid createdb testdb en_US  
+   * cubrid loaddb -u dba -d mdb_objects -s mdb_schema -i mdb_indexes  testdb
+     It‘s difference between sql and medium,  there is a unload file cubrid-testcases/medium/files/mdb.tar.gz that is used to medium test.(The files mdb_indexes,mdb_schema and mdb_objects are in the mdb.tar.gz archive)
+   * cubrid server start testdb
+   * csql -u dba testdb  
+   
+## 3.2 automatic monitoring configuration  
+**Test Machines**  
+
+No|role|user|ip|hostname|QUEUE_NAME|run_name
```

Comment:
Here Test Machines information is not useful.
About how to send a message and fire a test, move to CTP Introduction chapter.